staking-pool/src/lib.rs

```diff
@@ -183,6 +183,10 @@ impl StakingContract {
             "The owner account ID is invalid"
         );
         let account_balance = env::account_balance();
+        assert!(
```

Comment:
avoid `total_staked_balance`=0

Reply:
Realistically, this is impossible, because the storage required for the contract itself covers this.

Reply:
I see. The contract will stop working way before that.
Question: Who pays the storage fees? It comes from the staked NEARs?